fml/task_queue_id.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FML_TASK_QUEUE_ID_H_
+#define FLUTTER_FML_TASK_QUEUE_ID_H_
+
+#include "flutter/fml/logging.h"
+
+namespace fml {
+
+class TaskQueueId {
```

Comment:
Can we document his class as well as a description of what `kUnmerged` means.

Reply:
Done!